lib/slack/responses/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('../util/require-directory')(__dirname);
```

Comment:
Whilst using this file is fine, if we swapped to requireDir it would save us writing our own tests.